<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWidget</class>
 <widget class="QWidget" name="CameraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_Opciones">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Opciones</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>241</width>
     <height>451</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>241</width>
     <height>451</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>241</width>
     <height>451</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Resumen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_NumeroCuadros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Nº Cuadros: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_UsuarioId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario Id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_NumeroImagenes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nº Imágenes: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_CamaraId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cámara Id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_CalibracionId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibración Id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_SizeCuadros">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>T Cuadros:</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_lb_UsuarioId">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_lb_CalibrationId">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_lb_CamaraId">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_lb_NumeroCuadros">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_lb_SizeCuadros">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_lb_NumeroImagenes">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_lb_NumeroVista">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_NumeroVista">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vista Nº: </string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_Titulo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(52, 38, 38)
</string>
   </property>
   <property name="text">
    <string>CÁMARA - Obtención de vistas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Capturar">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>540</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Capturar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_imageCamera">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
border-width: 1.2px;
border-style:inset;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
